build.gradle

```diff
@@ -114,20 +117,13 @@ task copyMasterSheets(type: Copy) {
     include 'psheet_fantasy_std.htm'
     from 'outputsheets/d20/fantasy/htmlxml'
     into 'outputsheets/d20/historical/htmlxml'
-
-    into ('../../horror/htmlxml') {
-	    from 'outputsheets/d20/fantasy/htmlxml'
-    }
-    into ('../../sciencefiction/htmlxml') {
-	    from 'outputsheets/d20/fantasy/htmlxml'
-    }
-    into ('../../western/htmlxml') {
-	    from 'outputsheets/d20/fantasy/htmlxml'
-    }
+    into 'outputsheets/d20/horror/htmlxml'
```

Comment:
Have these changes been tested? Does the copymastersheets still refresh the sheets in these folders?